### PR TITLE
Use `factory_bot , ~> 6.3`

### PR DESCRIPTION
**Pull request by ydah**
This PR bump factory_bot to 6.3.

### CODE REVIEW

**Comment by dan-manges** [comment]
If `factory_bot` follows semver, should it be safe to change this to `~> 6.3` to only pin to the major version? Or does `factory_bot_rails` want to stay in sync with `factory_bot` by matching the minor version?

**Comment by ansonhoyt** [comment]
@ydah I noticed this has been bumped with https://github.com/thoughtbot/factory_bot_rails/pull/419

File: factory_bot_rails.gemspec [hunk 1]
```diff
@@ -14,6 +14,6 @@ Gem::Specification.new do |s|
   s.executables = []
   s.license = "MIT"
 
-  s.add_runtime_dependency("factory_bot", "~> 6.2.0")
+  s.add_runtime_dependency("factory_bot", "~> 6.3.0")
```
Comment (mathieujobin) [inline factory_bot_rails.gemspec:17]: `~> 6.2` sounds better to me

Comment (ydah) [reply factory_bot_rails.gemspec]: I updated this PR.

**Pull request closed without merging**